### PR TITLE
added gardenctl-v2 scripts

### PR DESCRIPTION
**Pull request by grolu**
updated github workflow actions

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by neo-liang-sap — APPROVED** [review]

File: .github/workflows/update-tags.yml [hunk 1]
```diff
@@ -23,6 +23,28 @@ jobs:
...
+          echo "Mac gardenctl-v2 binary hash '${{ github.event.client_payload.mac_sha }}'"
+          echo "Linux gardenctl-v2 binary hash '${{ github.event.client_payload.linux_sha }}'"
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # pin@v2.4.0
+      - name: Run update script
+        run:  ./.github/workflows/update-gardenctl-v2.sh ${{ github.event.client_payload.tag }} ${{ github.event.client_payload.darwin_sha }} ${{ github.event.client_payload.linux_sha }}
+      - name: Commit files
+        run: |
+          git config --local user.email "gardener.ci.user@gmail.com"
+          git config --local user.name "gardener-robot-ci-1"
+          git commit -m "Update gardenctl-v2" -a
```
Comment (petersutter) [inline .github/workflows/update-tags.yml:42]:
```suggestion
          git add gardenctl-v2.rb
          git commit -m "Update gardenctl-v2" -a
```
then you do not need to commit an empty `gardenctl-v2.rb` file

File: .github/workflows/update-tags.yml [hunk 2]
```diff
@@ -23,6 +23,28 @@ jobs:
...
           github_token: ${{ secrets.GITHUB_TOKEN }}
           force: false
+  update-gardenctl-v2:
+    if: github.event.client_payload.component == 'gardenctl-v2'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Event Information
+        run: |
+          echo "Event '${{ github.event.action }}' payload '${{ github.event.client_payload.tag }}'"
+          echo "Mac gardenctl-v2 binary hash '${{ github.event.client_payload.mac_sha }}'"
```
Comment (petersutter) [inline .github/workflows/update-tags.yml]:
```suggestion
          echo "Mac gardenctl-v2 binary hash '${{ github.event.client_payload.darwin_sha }}'"
```

File: .github/workflows/update-gardenctl-v2.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  def install
+    bin.install stable.url.split("/")[-1] => "gardenctl-v2"
+  end
+
+  test do
+    system "#{bin}/gardenctl-v2", "version"
+  end
+end
+
+EOF
```
Comment (petersutter) [inline .github/workflows/update-gardenctl-v2.sh:51]:
```suggestion
EOF

```

File: .github/workflows/update-gardenctl-v2.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+tag=${1}
+mac_sha=${2}
```
Comment (petersutter) [inline .github/workflows/update-gardenctl-v2.sh]: darwin_sha

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**